### PR TITLE
feat(VWO): add campaignName and variationId in properties

### PR DESCRIPTION
**Pull request by ItsSudip**
## PR Description

We are adding campaignName and variationId in properties of the track call we are getting through callback

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by yashasvibajpai — APPROVED** [review]

File: src/integrations/VWO/browser.js [hunk 1]
```diff
@@ -122,6 +122,8 @@ class VWO {
               this.analytics.track('Experiment Viewed', {
                 experimentId: expId,
                 variationName: _vwo_exp[expId].comb_n[variationId],
+                CampaignName: _vwo_exp[expId].name,
```
Comment (yashasvibajpai) [inline src/integrations/VWO/browser.js:125]: It is coming as `undefined` in sonarcloud analysis. Any issues?

Comment (ItsSudip) [reply src/integrations/VWO/browser.js:125]: _vwo_exp will be defined when vwo SDK will be loaded.

**Review by yashasvibajpai — DISMISSED** [review]

**Review by ujjwal-ab — APPROVED** [review]

File: .size-limit.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ module.exports = [
     name: 'Core - CDN',
     path: 'dist/legacy/rudder-analytics.min.js',
     gzip: true,
-    limit: '37.1 kB',
+    limit: '41 kB',
```
Comment (MoumitaM) [inline .size-limit.js:10]: There is no change in the core SDK still, why did you change these values?

**Review by MoumitaM — CHANGES_REQUESTED** [review]
Please revert the change or use appropriate size

File: .size-limit.js (hunk 2 above)
Comment (ItsSudip) [reply .size-limit.js:10]: I have not updated the packages so it was getting increased. Fixed that.

**Review by MoumitaM — APPROVED** [review]

**Review by MoumitaM — APPROVED** [review]

**Review by ujjwal-ab — APPROVED** [review]

**Review by yashasvibajpai — DISMISSED** [review]

**Review by MoumitaM — APPROVED** [review]

**Review by MoumitaM — APPROVED** [review]
